src/parser/diffeq_context.cpp

```diff
@@ -151,7 +151,7 @@ std::string DiffEqContext::get_non_cnexp_solution() const {
     std::string result;
     if (!deriv_invalid) {
         result = get_cvode_linear_diffeq();
-    } else if (deriv_invalid and eqn_invalid) {
+    } else if (deriv_invalid && eqn_invalid) {
```

Comment:
I thought I never used `and` instead of `&&`! Anyway, I guess this was not causing issue on windows?

Reply:
It was 😅